webapp/sources/rudder/rudder-web/src/main/resources/configuration.properties.sample

```diff
@@ -73,7 +73,7 @@ rudder.server.hsts=false
 # You MUST NOT enable it if you rely on plain HTTP for other sites or applications
 # on any (present or future) subdomain of you Rudder server's domain.
 #
-rudder.server.hsts.includeSubDomains=false
+rudder.server.hstsIncludeSubDomains=false
```

Comment:
I would have preferred 

```
rudder.server.hsts.enable = ...
rudder.server.hsts.includeSubDomains = ...
```

but I don't think it is worth the renaming/migration effort.

Reply:
Ah yes, it breaks because we have `rudder.server.hsts` too, I missed that.